packages/nimble-components/src/table/template.ts

```diff
@@ -20,7 +20,13 @@ import {
 // prettier-ignore
 export const template = html<Table>`
     <template role="table" ${children({ property: 'childItems', filter: elements() })}>
-        <div class="table-container">
+        <div class="table-container" style="
+            --ni-private-table-scroll-x: -${x => x.scrollX}px;
+            --ni-private-header-scrollbar-spacer-width: ${x => x.virtualizer.headerContainerMarginRight}px;
```

Comment:
nit: Since custom properties are global let's add `table` to scope it. 
```suggestion
            --ni-private-table-header-scrollbar-spacer-width: ${x => x.virtualizer.headerContainerMarginRight}px;
```